content/departments/engineering/dev/admin-exp/devops/managed/v1.1/mi1-1_upgrade_process.md

```diff
@@ -24,6 +24,14 @@ GOBIN=~/.bin make install
 
 ## Steps
 
+### Ensure we have release a new version of the managed_instance terraform module in `sourcegraph/deploy-sourcegraph-managed`
```

Comment:
@michaellzc I have few suggestions here:
1. Should we explicit said that these steps + golden update has to be done always with new release? and even put it as required step in [release process](https://handbook.sourcegraph.com/departments/engineering/dev/admin-exp/devops/managed/#release-process)?
2. Instead of writing this as 1-2 sentences, can we make it more 'engineer-friedly' so in points? :)
3. Is this something Cloud devops do or Erik? 
4. Could we add this steps as optional in [upgrade flow](https://handbook.sourcegraph.com/departments/engineering/dev/admin-exp/devops/managed/v1.1/mi1-1_upgrade_process/#upgrade)? I am afraid if this is only in 'ensure part', we might miss it for some customers.

Reply:
yes when there's something manual to be done I'll definitely let you guys know. For now, we can just assume that version updates just require bumping the version string and reapplying.